wlm/src/main/java/ai/djl/serving/wlm/ModelInfo.java

```diff
@@ -523,11 +523,20 @@ public void initialize() throws IOException, ModelException {
             }
         }
 
-        loadServingProperties();
-        downloadS3();
-        eventManager.onModelDownloaded(this, downloadDir);
-        configPerModelSettings();
-        downloadDraftModel();
+        if (SecureModeUtils.isSecureMode()) {
```

Comment:
Should network isolation enforced by hosting service?

Reply:
Yes, the network isolation will come from Hosting plane. So model files will always be mounted locally at container startup